src/main/java/uk/gov/pay/directdebit/payments/dao/PaymentViewDao.java

```diff
@@ -38,13 +37,13 @@ public PaymentViewDao(Jdbi jdbi) {
     }
 
     public List<PaymentView> searchPaymentView(PaymentViewSearchParams searchParams) {
+        
         String searchExtraFields = searchParams.generateQuery();
-        queryString = queryString.replace(":searchExtraFields", searchExtraFields);
         return jdbi.withHandle(handle -> {
-            Query query = handle.createQuery(queryString);
+            Query query = handle.createQuery(QUERY_STRING.replace(":searchExtraFields", searchExtraFields));
             Map<String, Object> queryMap = searchParams.getQueryMap();
-            Set<String> keys = queryMap.keySet();
-            keys.forEach(key -> query.bind(key, queryMap.get(key)));
+            queryMap.entrySet().forEach(value -> System.out.println(value));
```

Comment:
Not sure why we do `System.out.println(value)` ?

Reply:
Used for debug, will remove it 👍